docs/source/users/index.md

```diff
@@ -765,6 +765,16 @@ The `--response-path` option is a [JSONPath](https://goessner.net/articles/JsonP
 You can specify an allowlist, to only allow only a certain list of providers, or
 a blocklist, to block some providers.
 
+### Initializing default model providers
+
+This configuration allows for setting a default model, and embedding provider with their corresponding API keys.
+
+Following command line arguments can be used to initialize Jupyter AI extension with default providers:
+1. ```--AiExtension.default_model_provider_id```: Specify the default LLM model. E.g.: ```--AiExtension.default_model_provider_id="bedrock-chat:anthropic.claude-v2"```
+2. ```--AiExtension.default_embeddings_provider_id```: Specify default embedding provider. E.g.: ```--AiExtension.default_embeddings_provider_id="bedrock:amazon.titan-embed-text-v1"```
+3. ```--AiExtension.default_api_keys```: Specify model provider keys in a JSON format. E.g. ```--AiExtension.default_api_keys='{"OPENAI_API_KEY": "sk-abcd}'```
```

Comment:
```suggestion
When you start JupyterLab, you can use the following command-line arguments to set defaults for Jupyter AI:
1. ```--AiExtension.default_model_provider_id```: Specify the default language model. Example: ```--AiExtension.default_model_provider_id="bedrock-chat:anthropic.claude-v2"```
2. ```--AiExtension.default_embeddings_provider_id```: Specify default embedding provider. Example: ```--AiExtension.default_embeddings_provider_id="bedrock:amazon.titan-embed-text-v1"```
3. ```--AiExtension.default_api_keys```: Specify model provider keys in JSON format. E.g. ```--AiExtension.default_api_keys='{"OPENAI_API_KEY": "sk-abcd"}'```
```

"LLM" means "large language model", so "LLM model" suffers from [RAS syndrome](https://en.wikipedia.org/wiki/RAS_syndrome), the same affliction that one experiences when they enter their personal PIN number at the ATM machine to get $200 dollars of cash money. I tend to use "language model" generically where space isn't an issue.

AWS style, which I typically use in Jupyter AI's docs, discourages use of Latinisms, including commonly-used ones like e.g. and etc., to make text more readable for people for whom English is a second language. (There are a few of these in our docs already; I make an effort to remove them when I revise existing docs.)

Both of these first two settings seem misnamed: `default_model_provider_id` and `default_embeddings_provider_id` seem at first glance like they should be set to *provider IDs*, but they are actually set to *model IDs*, which include both a provider and a local model ID. Perhaps you could change them to `default_language_model` and `default_embedding_model`.

Reply:
Agreed! Making naming change. Also, replacing `E.g.` with `Sample CLI argument:`.

Reply:
Although the original names map directly to the names in JAI's config, I agree with the naming suggestion here. Can further simplify by dropping the default keyword. Update to `language_model` and  `embedding_model`. Updated the docs accordingly.